docs/Configuration.rst

```diff
@@ -1653,6 +1656,36 @@ The parameter is expressed in millisecond and the value default is
      # The delay when we consider a user as offline. Default value is 240000 milliseconds
     exo.user.status.offline.delay=240000
     
+.. _Configuration.sortField:    
+    
+=========================    
+Sort field configuration 
+=========================
+
+In :ref:`people application <People_Directory>` users are sorted to facilitate 
+the search in the directory.
+It is possible to choose the field on which to make the sort by defining this 
+parameter in :ref:`exo.properties <Configuration.ConfigurationOverview>` file:
+
+::
+
+	exo.social.identity.sort.field=fullname
+	
+.. note:: The sort is feasible on these fields: fullname, lastname or firstname.
```

Comment:
```suggestion
.. note:: The following values are accepted: fullname, lastname or firstname.
```

Reply:
I think we should explain where the full name is picked from. How different is it form what is displayed ?

Reply:
done